app/models/step_by_step_page.rb

```diff
@@ -49,6 +49,7 @@ def mark_as_published
     update_attribute(:draft_updated_at, now)
     update_attribute(:scheduled_at, nil)
     update_attribute(:assigned_to, nil)
+    update_attribute(:status, "published")
```

Comment:
Can all of these (and others from other methods) calls to `update_attribute` be rewritten one call to `update { published_at: now, draft_updated_at: now ... }`  saving a few database calls

Reply:
Updated and fixed-up